sdk/tables/azure-data-tables/src/tables_clients.cpp

```diff
@@ -135,6 +135,17 @@ TableServicesClient::TableServicesClient(
       std::move(perOperationPolicies2));
 }
 
+TableServicesClient::TableServicesClient(
```

Comment:
Is there more than one Table service? This class definition is `TableServicesClient`, with Services being plural. That implies that there is more than one service.

Reply:
done